### PR TITLE
Created StoreRepository and StoreService with two methods.

### PR DESCRIPTION
**Pull request by grigveronika**
1. findStoresByProfileId  - user can choose his stores
2. findStoresByProfileIdAndId - user can choose one of his stores

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/service/StoreService.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import internship.mbicycle.storify.model.Store;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface StoreService {
+    List<Store> findStoresByProfileId(Long profileId);
+
+    Optional<Store> findStoresByProfileIdAndId(Long profileId, Long id);
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/service/StoreService.java:11]: findStoresByIdAndProfileId

File: src/main/java/internship/mbicycle/storify/repository/StoreRepository.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import internship.mbicycle.storify.model.Store;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface StoreRepository extends JpaRepository<Store, Long> {
+    List<Store> findStoresByProfileId(Long profileId);
+    Optional<Store> findStoresByProfileIdAndId(Long profileId, Long id);
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/repository/StoreRepository.java:11]: findStoresByIdAndProfileId

File: src/main/java/internship/mbicycle/storify/model/Store.java [hunk 3]
```diff
@@ -34,10 +38,10 @@
 
     private Long profit;
 
-// нужна entity Profile
-//    @ManyToOne(cascade = {CascadeType.PERSIST, CascadeType.DETACH, CascadeType.REFRESH, CascadeType.MERGE})
-//    @JoinColumn(name = "profile_id")
-//    private Profile profile;
+    @JsonIgnore
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Store.java:41]: I'm not sure you really need this annotation

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/service/StoreService.java (hunk 1 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/service/StoreService.java:11]: changed name og method

File: src/main/java/internship/mbicycle/storify/repository/StoreRepository.java (hunk 2 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/repository/StoreRepository.java:11]: changed name of method

**Pull request merged**